examples/cryptocurrency-advanced/backend/src/lib.rs

```diff
@@ -14,7 +14,12 @@
 
 //! Cryptocurrency implementation example using [exonum](http://exonum.com/).
 
-#![deny(missing_debug_implementations, missing_docs, unsafe_code, bare_trait_objects)]
```

Comment:
It is funny because some time ago this code was written in the multiline form, but then we switched to a more compact representation. Personally I prefer the single line form. Additionally, it is probably a rare situation when such lines would be changed significantly and it isn't hard to see the difference, especially with coloring.